apps/vercel/src/components/config-screen/ContentTypeSelect.tsx

```diff
@@ -0,0 +1,68 @@
+import { ChangeEvent, Dispatch, SetStateAction, useEffect } from 'react';
+import { Box, FormControl, Select } from '@contentful/f36-components';
+import { styles } from './ConfigScreen.styles';
+import { AppInstallationParameters } from '../../types';
+import { actions } from '../parameterReducer';
+import { ConfigAppSDK } from '@contentful/app-sdk';
+
+const ContentTypeSelect = ({
+  parameters,
+  dispatch,
+  sdk,
+}: {
+  parameters: AppInstallationParameters;
+  dispatch: Dispatch<SetStateAction<any>>;
+  sdk: ConfigAppSDK;
+}) => {
+  useEffect(() => {
+    async function getContentTypes() {
+      const contentTypesResponse = await sdk.cma.contentType.getMany({});
```

Comment:
I wonder, should we inject the content types into this select vs. fetching them ever time? I think content types seldomly change and it might be more efficient to fetch them once on app config load.

(Feel free to tag for a follow up PR).